infra/aws/README.md

```
@@ -0,0 +1,250 @@
+
+CloudFront, Lambda, S3 등 AWS 네이티브 리소스를 정의하는 CloudFormation
+템플릿과 소스 코드를 모아둔 디렉토리입니다. Ansible 의 `image_cdn`
+역할이 배포를 담당합니다.
+
+---
+
+## 목차
+
+1. [디렉토리 구조](#1-디렉토리-구조)
+2. [Phase 1 — Image CDN](#2-phase-1--image-cdn)
+   - [아키텍처](#21-아키텍처)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)